JSTests/wasm/wasm.json

```diff
@@ -15,15 +15,15 @@
         "nullfuncref":   { "type": "varint7", "value":  -13, "b3type": "B3::Void",   "width": 0 },
         "nullexternref": { "type": "varint7", "value":  -14, "b3type": "B3::Void",   "width": 0 },
         "nullref":       { "type": "varint7", "value":  -15, "b3type": "B3::Void",   "width": 0 },
-        "funcref":       { "type": "varint7", "value":  -16, "b3type": "B3::Int64",  "width": 64 },
-        "externref":     { "type": "varint7", "value":  -17, "b3type": "B3::Int64",  "width": 64 },
+        "funcref":       { "type": "varint7", "value":  -16, "b3type": "(is32Bit() ? B3::Int32 : B3::Int64)",  "width": 64 },
+        "externref":     { "type": "varint7", "value":  -17, "b3type": "(is32Bit() ? B3::Int32 : B3::Int64)",  "width": 64 },
```

Comment:
Can we just make these `B3::pointerType()`?

Reply:
Ah, sweet!